apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -43,11 +66,24 @@ export function AiGeneratePage({ resLocals }: { resLocals: Record<string, any> }
                   Create question
                 </button>
               </form>
+              Or choose a question from our list of test questions:
```

Comment:
"test questions" has a connotation that I'd like to avoid:

```suggestion
              Or choose a question from our list of example prompts:
```

Let's refer to these the same elsewhere too. Currently you call them "prebuilt questions"; let's use "example prompts" elsewhere too.

Reply:
Done in 46fe88d71bb20bf3e2763ccd99edba1b5435ea38